core/native_type_traits.h

```diff
@@ -160,4 +160,43 @@ struct NativeTypeTraits<IDLString> : public NativeTypeTraitsBase<IDLString> {
   }
 };
 
+// FIXME(Hwansung): should be generated automatically in another file.
```

Comment:
Okay, for now as-is okay. But this is not exact NativeTypeTraits because the IDL enum type will always be string type in c++ side.